operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -92,13 +92,19 @@ Mattermost documentation covers a number of different topics. For documentation,
 
 **Dev Review**
 
-* Admin Guide - Enterprise: Scott Bishel \(@sbishel\)
-* Admin Guide - Non-enterprise: Joram Wilander \(@jwilander\)
-* End User Guide - Mobile and desktop app: Elias Nahum \(@enahum\)
-* End User Guide - Other: Dean Whillier \(@deanwhillier\)
+It's not always clear which developer should be added as a reviewer for a pull request. In some cases you may be prompted to add a reviewer based on them having recently viewed or edited the file you're submitting in the PR. If not, one of the team leads listed below can be added as a reviewer for documentation relevant to their area of expertise: 
```

Comment:
@justinegeffen What is the message/communication you intend to convey with this paragraph?

Reply:
I'm working on a reworded version now, good callout.

Reply:
I've reworded it and it should be clearer.